src/main/java/io/opencensus/integration/jdbc/Observability.java

```diff
@@ -224,33 +227,30 @@ public void close() {
         // Finally record the latency of the entire call.
         recordTaggedStat(KEY_METHOD, this.method, MEASURE_LATENCY_MS, timeSpentMs);
       } finally {
-        this.spanScope.close();
-        this.closed = true;
+        closed = true;
       }
     }
 
-    /*
-     * recordException annotates the underlying span with the description of the exception
-     * and also sets its status to indicate the exception but it also increments the number
-     * of errors by 1, adding tags: "method", "reason" to the recorded metric.
-     * */
-    void recordException(Exception e) {
+    // Ends and Annotates the underlying span with the description of the exception and also sets
+    // its status to indicate the exception but it also increments the number of errors by 1,
+    // adding tags: "method", "reason" to the recorded metric.
+    void endWithException(Exception e) {
       String detail = e.toString();
-      this.span.setStatus(Status.INTERNAL.withDescription(detail));
```

Comment:
Is removing `this` a style guide recommendation? I ask because I find it handy with reading complex code to know who owns what, and hence why I've been using it.